sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/security/CertificateNamesGenerator.java

```diff
@@ -41,15 +42,19 @@
     // ub-common-name INTEGER ::= 64
     private static final int CN_MAX_LENGTH = 64;
 
-    public CertificateNamesGenerator(String serviceName, TaskSpec taskSpec, PodInstance podInstance) {
+    public CertificateNamesGenerator(String serviceName,
```

Comment:
The same comment here as for `getTaskEnvironment ` w.r.t the ordering of the parameters.